source/docs/advanced/shipping/mondial-relay.md

```diff
@@ -21,44 +21,51 @@ Then, within your Front-Commerce project, you have to:
 
 - Use the module in your `.front-commerce.js`
   ```diff
-// .front-commerce.js
-module.exports = {
+  // .front-commerce.js
+  module.exports = {
   name: "Front Commerce DEV",
   url: "http://www.front-commerce.test",
   modules: [
-+   "./node_modules/front-commerce/modules/shipping-mondialrelay"
-    "./src"
+  ```
+
+* "./node_modules/front-commerce/modules/shipping-mondialrelay"
```

Comment:
Why are addition markers replaces with asterisks in this file ?

Reply:
I think it is because the `  ```diff` is indented and not the code below it… it triggered some weird fixes indeed.

Fixed manually in bc38563675bc4e83fee9c6aaa41060e395997968